.travis.yml

```diff
@@ -43,7 +43,7 @@ env:
     - PATH=$USERDIR/bin:$PATH
     - LD_LIBRARY_PATH=$USERDIR/lib:$LD_LIBRARY_PATH
     - LEATHERMAN_VERSION=0.9.0
-    - CPPHOCON_VERSION=0.1.2
+    - CPPHOCON_VERSION=0.1.3
```

Comment:
Ahh ok..In that case perhaps a mention in the commit message as to why hocon is being bumped for a fact caching PR.